lib/Monitor.php

```diff
@@ -60,6 +60,18 @@ public static function put($apiKey, $apiVersion, $params = [])
         }
     }
 
+    public static function getYaml($apiKey, $apiVersion)
+    {
+        $client = self::getMonitorHttpClient($apiKey, $apiVersion);
+        $response = $client->get('.yaml', ['timeout' => 10]);
```

Comment:
we might want to allow a slightly longer timeout -- users with a large number of monitors (>100) might not actually get a file generated in 10s currently 😬

Reply:
Good point! Even a small config was taking 2-3 seconds to fully generate and download. I set to 25 seconds. I think the Python library is set at 10 seconds FYI.